editor/editor_autoload_settings.cpp

```diff
@@ -33,6 +33,7 @@
 #include "core/global_constants.h"
 #include "core/project_settings.h"
 #include "editor_node.h"
+#include "modules/gdscript/gdscript.h"
```

Comment:
We shouldn't have dependency on modules in `editor/` normally, as you can compile the engine with `module_gdscript_enabled=no`. So at least there should be `#ifdef GDSCRIPT_ENABLED` guards, but ideally this should call to the abstract ScriptLanguage API somehow.

Reply:
Ah.. didnt knew that its possible and used somehow. GDScript is such a core engine element so its hardly to even present in mind that Godot can be used without it.